VerifierApp.Api/VerifierApp.Api/Services/DeskBookingService.cs

```
@@ -32,28 +32,36 @@ public async Task<List<EmployeeDeskBooking>> GetEmployeeDeskBookingsByDate(DateT
             var dataRange = $"'{sheetName}'!{range.FormattedRange}";
 
             var employeeBookings = new List<EmployeeDeskBooking>();
-            var response = await _googleSheetsService.ReadValuesByRange(dataRange);
 
-            if (response.Values?.Count > 0)
+            try
             {
-                for (var currentRow = 0; currentRow < response.Values.Count; currentRow++)
+                var response = await _googleSheetsService.ReadValuesByRange(dataRange);
+
+                if (response.Values?.Count > 0)
                 {
-                    var employeeName = response.Values[currentRow][0].ToString();
-                    if (!string.IsNullOrEmpty(employeeName))
+                    for (var currentRow = 0; currentRow < response.Values.Count; currentRow++)
                     {
-                        employeeBookings.Add(
-                            new EmployeeDeskBooking(employeeName, int.Parse(response.Values[currentRow][1].ToString() ?? string.Empty), 
-                                new CellLocation(sheetName, range.Start.ColumnLetter, range.Start.RowNumber + currentRow)));
+                        var employeeName = response.Values[currentRow][0].ToString();
+                        if (!string.IsNullOrEmpty(employeeName))
+                        {
+                            employeeBookings.Add(
+                                new EmployeeDeskBooking(employeeName, int.Parse(response.Values[currentRow][1].ToString() ?? string.Empty),
+                                    new CellLocation(sheetName, range.Start.ColumnLetter, range.Start.RowNumber + currentRow)));
+                        }
                     }
                 }
             }
+            catch (Exception)
+            {
+                return new List<EmployeeDeskBooking>();
```

Should this throw an exception rather than returning an empty list?  If not, is it at risk of just being silently swallowed?